AWS EMF Exporter failing test

**Describe the bug**
For some reason I've not dived in to fully diagnose, when running on mac os a [recently introduced test](https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/434bfdaacd43af8791704f65c65273db42f8e453#diff-62696b712af68a46d758e5e68db36277R59) is consistently failing (@shaochengwang).

**Steps to reproduce**
```
cd exporter/awsemfexporter
go test -run=TestConsumeMetrics
```

**What did you expect to see?**
Passing test

**What did you see instead?**
```
go test -run=TestConsumeMetrics
--- FAIL: TestConsumeMetrics (1.65s)
    emf_exporter_test.go:114:
        	Error Trace:	emf_exporter_test.go:114
        	Error:      	An error is expected but got nil.
        	Test:       	TestConsumeMetrics
FAIL
exit status 1
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsemfexporter	2.202s
```

**What version did you use?**
e3493cedf4b6f7419be7e4e61b4186e51073f411

**What config did you use?**
n/a

**Environment**
OS: mac os 10.15.6
Compiler(if manually compiled): go1.14 darwin/amd64

**Additional context**
Walking through briefly w/ goland I see that there is an (expected?) `cloudwatchlogs.InvalidSequenceTokenException` error followed by no error on successful retry.

For context the error that is expected by the test but not raised locally is:

```
--- FAIL: TestConsumeMetrics (9.46s)
    emf_exporter_test.go:114: 
        	Error Trace:	emf_exporter_test.go:114
        	Error:      	Received unexpected error:
        	            	NoCredentialProviders: no valid providers in chain. Deprecated.
        	            		For verbose messaging see aws.Config.CredentialsChainVerboseErrors
        	Test:       	TestConsumeMetrics
```

- from https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-collector-contrib/4477/workflows/03afdd10-8afd-4db0-a15b-ebaa19dabe93/jobs/32280

--- a/exporter/awsemfexporter/emf_exporter_test.go
+++ b/exporter/awsemfexporter/emf_exporter_test.go
@@ -34,6 +34,11 @@ import (
 	"go.uber.org/zap"
 )
 
+func init() {
+	os.Setenv("AWS_ACCESS_KEY_ID", "test")
+	os.Setenv("AWS_SECRET_ACCESS_KEY", "test")
+}
+
 type mockPusher struct {
 	mock.Mock
 }

